src/GitVersionCore.Tests/Mocks/MockRepository.cs

```diff
@@ -1,5 +1,8 @@
 using System;
 using System.Collections.Generic;
+
+using GitVersionCore.Tests.Mocks;
```

Comment:
The `Mock*` classes are all without namespace. I accidentally created the new ones with the sub-namespace `Mock`, but will fix this later.